server/bgcompute.py

```diff
@@ -97,6 +99,25 @@ def bgcompute_update_ballot_manifest_file() -> int:
     return len(files)
 
 
+def bgcompute_update_batch_tallies_file() -> int:
+    files = (
+        File.query.join(Jurisdiction, File.id == Jurisdiction.batch_tallies_file_id)
+        .filter(File.processing_started_at.is_(None))
+        .all()
+    )
+
+    for file in files:
+        try:
+            jurisdiction = Jurisdiction.query.filter_by(
+                batch_tallies_file_id=file.id
+            ).one()
+            process_batch_tallies_file(db_session, jurisdiction, file)
+        except Exception:
+            print("ERROR updating batch tallies file")
```

Comment:
worth logging the jurisdiction ID for debugging?

Reply:
this is a great question. in general I was confused by the pattern of swallowing the exception in this file, but I followed it assuming there was some thinking behind it.

my gut says we should log some context (like the jurisdiction id) and the exception itself. thoughts?

Reply:
yeah that sounds good.

Reply:
coming back to this in a separate PR